packages/react/src/components/Authenticator/FederatedIdentities/context/ProviderDataListContext.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  FederatedProviderList,
+  ProviderData,
+  ProviderType,
+} from '../controls/types';
+import { FederatedProvider } from '@aws-amplify/ui';
+
+function capitalizeFirstLetter(str: string) {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
```

Comment:
I believe we have a util for this already - https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/utils/utils.ts#L153